mailet/base/src/test/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -211,19 +253,36 @@ public void removeAttribute(String name) {
     }
 
     public void sendMail(MimeMessage mimemessage) throws MessagingException {
```

Comment:
you defeat builder's purpose with these factory methods. Why don't you fix the callers ?